packages/core/src/odata-v4/request-builder/create-request-builder.ts

```diff
@@ -89,18 +52,6 @@ export class CreateRequestBuilderV4<EntityT extends EntityV4>
     destination: Destination | DestinationNameAndJwt,
     options?: DestinationOptions
   ): Promise<EntityT> {
-    this.prepare();
-    return this.build(destination, options)
-      .then(request => request.execute())
-      .then(response =>
-        deserializeEntityV4(
-          getSingleResult(response.data),
-          this._entityConstructor,
-          response.headers
-        )
-      )
-      .catch(error =>
-        Promise.reject(errorWithCause('Create request failed!', error))
-      );
+    return super.execute(destination, options);
```

Comment:
The `execute()` method in this file can be removed.

Reply:
ah good catch. Removed.